test/internet/test-dns-ipv6.js

```diff
@@ -44,46 +40,49 @@ function checkWrap(req) {
 }
 
 TEST(function test_resolve6(done) {
-  var req = dns.resolve6('ipv6.google.com', function(err, ips) {
-    if (err) throw err;
+  const req = dns.resolve6('ipv6.google.com',
+    common.mustCall((err, ips) => {
+      assert.ifError(err);
 
-    assert.ok(ips.length > 0);
+      assert.ok(ips.length > 0);
 
-    for (var i = 0; i < ips.length; i++) {
-      assert.ok(isIPv6(ips[i]));
-    }
+      for (let i = 0; i < ips.length; i++) {
+        assert.ok(isIPv6(ips[i]));
+      }
 
-    done();
-  });
+      done();
+    }));
 
   checkWrap(req);
 });
 
 TEST(function test_reverse_ipv6(done) {
-  var req = dns.reverse('2001:4860:4860::8888', function(err, domains) {
-    if (err) throw err;
+  const req = dns.reverse('2001:4860:4860::8888',
+    common.mustCall((err, domains) => {
+      assert.ifError(err);
 
-    assert.ok(domains.length > 0);
+      assert.ok(domains.length > 0);
 
-    for (var i = 0; i < domains.length; i++) {
-      assert.ok(domains[i]);
-      assert.ok(typeof domains[i] === 'string');
-    }
+      for (let i = 0; i < domains.length; i++) {
+        assert.ok(domains[i]);
+        assert.ok(typeof domains[i] === 'string');
+      }
 
-    done();
-  });
+      done();
+    }));
 
   checkWrap(req);
 });
 
 TEST(function test_lookup_ipv6_explicit(done) {
-  var req = dns.lookup('ipv6.google.com', 6, function(err, ip, family) {
-    if (err) throw err;
-    assert.ok(net.isIPv6(ip));
-    assert.strictEqual(family, 6);
+  const req = dns.lookup('ipv6.google.com', 6,
+    common.mustCall((err, ip, family) => {
+      assert.ifError(err);
+      assert.ok(net.isIPv6(ip));
```

Comment:
Nit: `isIPv6` is already defined at the beginning of the file. We can use that consistently.